### ISSUE TITLE
["BUG"]Upgrade to Nutmeg: No codejailservice found in docker.io with tag 14.0.0

### ISSUE THREAD

**Issue by insad** [eduNEXT/tutor-contrib-codejail#23]
The nutmeg version can't find codejailservice with tag 14.0.0 in the docker registry:

```
Pulling codejailservice (docker.io/ednxops/codejailservice:14.0.0)...
ERROR: manifest for ednxops/codejailservice:14.0.0 not found: manifest unknown: manifest unknown
Error: Command failed with status 1: docker-compose -f /srv/tutor/env/local/docker-compose.yml -f /srv/tutor/env/local/docker-compose.prod.yml -f /srv/tutor/env/local/docker-compose.tmp.yml --project-name tutor_local up --remove-orphans --build -d
```

![image](https://user-images.githubusercontent.com/3638794/173186648-720a456c-6f27-435b-84c1-8fa4f417def9.png)

**Comment by mariajgrimaldi** [comment]
We just uploaded the image 14.0.0 to the repository and released the plugin with its 14.0.0 version. Can you test if it works for you?

**Comment by insad** [comment]
Works perfectly now!
